src/DIRAC/WorkloadManagementSystem/Service/JobMonitoringHandler.py

```diff
@@ -70,10 +70,9 @@ def initializeHandler(cls, svcInfoDict):
 
     return S_OK()
 
-  def initialize(self):
+  def initializeRequest(self):
```

Comment:
I'm surprised this was passing the integration tests, is it simply that we don't have test coverage for this or is there something wrong with the HTTPS test setup?

Reply:
I want to verify that too, I will try running locally.